src/test/java/gov/healthit/chpl/aqa/stepDefinitions/SurveillanceComplaintsSteps.java

```diff
@@ -0,0 +1,139 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.SurveillanceComplaintsPage;
+
+/**
+ * Class SurveillanceComplaintsSteps definition.
+ */
+public class SurveillanceComplaintsSteps extends Base {
+    @Given("^I am on complaints page$")
+    public void loadComplaintsPage() {
+        getDriver().get(getUrl() + "#/surveillance/complaints");
+    }
+
+    @When("^I click Add New Complaint button$")
+    public void clickAddNewComplaintButton() {
+      SurveillanceComplaintsPage.addNewComplaintButton(getDriver()).click();
+    }
+
+    /**
+     * Fill data in complaint form.
+     * @param acb is ACB for which complaint is being entered
+     * @param receivedDate is received date of complaint
+     * @param closedDate is date when complaint was closed
+     * @param acbComplaintId ACB complaint ID
+     * @param oncComplaintId ONC complaint ID
+     * @param complainantType is type of complainant
+     * @param summary is summary of a complaint
+     */
+    @When("^I enter complaint data in fields: \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\"$")
+    public void enterDataInComplaintFields(final String acb, final String receivedDate, final String closedDate,
+            final String acbComplaintId, final String oncComplaintId, final String complainantType,
+            final String summary) {
+        SurveillanceComplaintsPage.oncACBField(getDriver()).sendKeys(acb);
+        SurveillanceComplaintsPage.receivedDate(getDriver()).sendKeys(receivedDate);
+        SurveillanceComplaintsPage.closedDate(getDriver()).sendKeys(closedDate);
+        SurveillanceComplaintsPage.oncACBComplaintId(getDriver()).sendKeys(acbComplaintId);
+        SurveillanceComplaintsPage.oncComplaintId(getDriver()).sendKeys(oncComplaintId);
+        SurveillanceComplaintsPage.complainantType(getDriver()).sendKeys(complainantType);
+        SurveillanceComplaintsPage.summary(getDriver()).sendKeys(summary);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", SurveillanceComplaintsPage.complainantContactedCheckbox(getDriver()));
```

Comment:
I see that you're checking all of these checkboxes, but there was no boolean in the arguments as to whether they should be selected or not. I think I'd prefer not selecting these at all unless there is an explicit request to do so as an argument

Reply:
Updated test to not select checkboxes. Also, updates are expected to these fields in UI review work